docs/050-breaking-changes.rst

```diff
@@ -0,0 +1,380 @@
+********************************
+Solidity v0.5.0 Breaking Changes
+********************************
+
+This section highlights the main breaking changes introduced in Solidity
+version 0.5.0, along with the reasoning behind the changes and how to update
+affected code.
+For the full list check
+`the release changelog <https://github.com/ethereum/solidity/releases/tag/v0.5.0>`_.
+
+Semantic Only Changes
+=====================
+
+This section lists the changes that are semantic-only, thus potentially
+hiding new and different behaviors in previous code.
+
+* Signed right shift now uses proper arithmetic shift, i.e. rounding towards
+  negative infinity. Signed and unsigned shift will have dedicated opcodes in
+  Constantinople, and are emulated by Solidity for the moment.
+
+* The ``continue`` statement in a ``do...while`` loop now jumps to the
+  condition, which is the common behavior in such cases. It used to jump to the
+  loop body. Thus, if the condition is false, the loop terminates.
+
+* The functions ``.call()``, ``.callcode()``, ``.delegatecall()`` and
+  ``.staticcall()`` do not pad anymore when given a single ``bytes`` parameter.
+
+* Pure and view functions are now called using the opcode ``STATICCALL``
+  instead of ``CALL``, which disallows state changes on the EVM level.
+
+* The ABI encoder now properly pads data from calldata (``msg.data`` and
```

Comment:
Perhaps clarify by: `... properly pads byte arrays and strings from calldata (...) when used in external function calls and in abi.encode.`